terraform/panther_cloud_pubsub_transport_type_infra/production.tfvars

```diff
@@ -1,6 +1,9 @@
-project_id                           = "project-id"
-topic_name                           = "topic-name"
-subscription_id                      = "subscription-id"
-gcp_region                           = "gcp-region"
+# Basic Info
+project_id      = "project-id"      # ID (name) of your GCP project
+topic_name      = "topic-name"      # Topic name to be created. Data will need to be published here
+subscription_id = "subscription-id" # Name for the subscription panther will use to consume from the topic
+gcp_region      = "gcp-region"      # Region of the topic/subscription tuple
+
+# Panther needs a Service account to be able to authenticate and pull data from the topic
```

Comment:
Is a `Service account` a common term in Google Cloud or maybe we should include a link here?

Reply:
It is a common term, yes, but I can add one.

Reply:
Actually we have that in the docs. I don't think it's worth it.